src/NServiceBus.Testing.Tests/NServiceBus.Testing.Tests.csproj

```diff
@@ -10,15 +10,10 @@
 
   <ItemGroup>
     <PackageReference Include="ApprovalTests" Version="3.0.13" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
     <PackageReference Include="Mono.Cecil" Version="0.10.0-beta6" />
     <PackageReference Include="NUnit" Version="3.7.1" />
-    <PackageReference Include="NUnit3TestAdapter" Version="3.7.0" />
+    <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
```

Comment:
not sure whether it solves anything tbh. I was mostly trying to copy the core's test settings as the approval test doesn't seem to run at all in this project.